examples/kserve_deployment/README.md

```diff
@@ -402,7 +481,7 @@ Moreover, ZenML will manage the KServe deployments inside the same `kubeflow`
 namespace where the Kubeflow pipelines are running. You also have to update the 
 set of permissions granted by Kubeflow to the Kubernetes service account in the 
 context of which Kubeflow pipelines are running to allow the ZenML workloads to 
-create, update and delete KServe InferenceServices, Secrets and ServiceAccounts. 
+create, update and delete KServe InferenceServices, KServes and ServiceAccounts. 
```

Comment:
does KServe have some custom resources with the name `KServes`?

Reply:
LOL